client/my-sites/upgrades/cart/cart-item.jsx

```diff
@@ -113,7 +113,7 @@ const CartItem = React.createClass( {
 		} else if ( getIncludedDomain( this.props.cartItem ) ) {
 			info = getIncludedDomain( this.props.cartItem );
 		} else if ( isTheme( this.props.cartItem ) ) {
-			info = this.props.selectedSite.domain;
+			info = this.props.selectedSite && this.props.selectedSite.domain;
```

Comment:
`domain` is set to `this.props.cartItem.meta`, I'm not sure that this is desirable in that case,
otherwise we could just drop that `if`, and let it fall to the `else`